basics/task7/logout.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+// After destroying the task7 , it redirects to the login page.
+session_start();
+session_unset();
+session_destroy();
+header('Location: /');
+exit();
```

Comment:
Since we have already destroyed session and the user is also redirected, so do we need to use the exit function in this specific scenario ?